RepoDb.Core/RepoDb/ClassMap.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Linq.Expressions;
+using RepoDb.Interfaces;
+
+namespace RepoDb
+{ 
+    public class ClassMap<T> where T : class
+    {
+        public Type EntityType { get; }
+
+        public ClassMap()
+        {
+            EntityType = typeof(T);
+        }
+        
+        protected ClassMap<T> Table(string table)
+        {
+            ClassMapper.Add<T>(table);
+            return this;
+        }
+        
+        protected IIdentityOptions<T> Id(Expression<Func<T, object>> expression)
```

Comment:
Apology, can we make the name of this method `Identity`?

Reply:
Of course, you are the author, You can demand everything